wiki/es/es-Privacy-Statement.md

```diff
@@ -24,7 +24,7 @@ Cuando te conectas a un servidor público o privado, el operador del servidor pu
 
 Como operador de un servidor, cuando tu registras un servidor público con un Servidor Central, tu dirección IP es enviada a CloudFare (1.1.1.1) para identificar tu dirección IP pública. Las direcciones IP de todos los servidores públicos registrados con el Servidor Central también pueden ser vistas por terceros para fines informativos u otros fines (por ejemplo [aquí](http://jamulus.softins.co.uk/)). Tu dirección IP pública no es registrada ni guardada por parte de Jamulus, pero puede ser guardada o procesada por terceros.
 
-_Los usuarios preocupados por el uso de la red de CloudFare pueden [re-compilar](Compiling) el código fuente de Jamulus para utilizar una alternativa para WELL_KNOWN_HOST/ WELL_KNOWN_PORT en [global.h](https://github.com/corrados/jamulus/blob/master/src/global.h#L116)_
+_Los usuarios preocupados por el uso de la red de CloudFare pueden [re-compilar](Compiling) el código fuente de Jamulus para utilizar una alternativa para WELL_KNOWN_HOST/ WELL_KNOWN_PORT en [global.h](https://github.com/corrados/jamulus/blob/master/src/global.h#L111)_
```

Comment:
Has it changed in the english version too? If so, please open a pr for the english version on changes too.

Reply:
Will do.